lemonHope.py

```diff
@@ -1,136 +1,19 @@
-import asyncio
 import os
-import random
-import re
 
 from discord.ext import commands
 from dotenv import load_dotenv
-from tinydb import TinyDB, Query
 
-print('Lemon is starting')
-load_dotenv()
-token = os.getenv('lemonhope_token')
-saveLock = asyncio.Lock()
+import quotes.commands as quotes
 
-lemon = commands.Bot(command_prefix="Lemon, ")
+def run():
+    print('Lemon is starting')
+    load_dotenv()
+    token = os.getenv('lemonhope_token')
 
+    lemon = commands.Bot(command_prefix="Lemon, ")
 
-def getDBFromGuild(guild):
-    """
-    Retrieves db from guild name
-    """
-    return TinyDB(r'data/' + guild + r'.json')
+    quotes.setup(lemon)
```

Comment:
This feels a bit odd, though I haven't used cog before. Shouldn't bot be responsible from loading the commands instead of the commands for registering with bot?

Reply:
This is a paradigm I'm seeing a lot of. It allows the commands module to own it's own logic which is helpful if it's doing more complex stuff. For example, if it needs to register threads and other events with the bot it can own that. This becomes significantly cleaner and easier to build out in the future.